VmAgent.Core/VmPathHelper.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Gaming.VmAgent.Core
+{
+    using AgentInterfaces;
+    using ApplicationInsights;
+    using ApplicationInsights.Extensibility;
+    using Microsoft.Extensions.Logging;
+    using VmAgent.Core;
+
+    public static class VmPathHelper
```

Comment:
Should this class just belong in the LocalMultiplayerAgent project. It doesn't relate to the Core package since it doesnt apply for PlayFab hosted VMs

Reply:
The reason I moved that code from LocalMultiplayerAgent to Core is I created unit tests to test methods in Core library for LocalMultiplayerAgent (SessionHostContainerConfigurationUnitTests.cs), then what is another way to test LocalMultiplayerAgent behavior when it calls methods in core libraries? Should I move test file location under LocalMultiplayerAgent ?

Reply:
Yeah, I guess if you need to use it there, then there is no choice.